server/src/completions/overrides.ts

```diff
@@ -0,0 +1,20 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) 2023 Savoir-faire Linux. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+export const OVERRIDES = [
+  'append',
```

Comment:
append, prepend and remove are not technically overrides, they are operators. However they are used in a similar fashion.

Reply:
As discussed, I removed everything but `append`, `prepend`, and 'remove`. Await your progress to continue on the overrides